index.html

```diff
@@ -55,21 +55,17 @@
     <banner class="banner">
```

Comment:
Additional: 
1. missing color in header-nav-link
![image](https://user-images.githubusercontent.com/47839903/206339468-8e638048-9949-4a6d-aa0d-5ec98af4b901.png)

2. merge all commits into 1 before push